scripts/zones/Aht_Urhgan_Whitegate/npcs/Porter_Moogle.lua

```diff
@@ -0,0 +1,59 @@
+-----------------------------------
+--  Area:   Aht Urhgan Whitegate
+--  NPC:    Porter Moogle
+--  Type:   Storage Moogle
+--  @zone 50
+--  @pos TODO
+-----------------------------------
+package.loaded["scripts/zones/Aht_Urhgan_Whitegate/TextIDs"] = nil;
+-----------------------------------
+
+require("scripts/zones/Aht_Urhgan_Whitegate/TextIDs");
+require("scripts/globals/porter_moogle_util");
+
+local e = 
+{
+    TALK_EVENT_ID       =   957,
+    STORE_EVENT_ID      =   958,
+    RETRIEVE_EVENT_ID   =   959,
+    ALREADY_STORED_ID   =   960,
+    MAGIAN_TRIAL_ID     =   963
+};
+
+RETRIEVE_DIALOG_ID  =   13462;
```

Comment:
should probably move this to the zone's TextIDs.lua, and comment the message it displays (for compatibility with the batch is updates)